contrib/zenith/libpagestore.c

```diff
@@ -34,23 +34,23 @@ void		_PG_init(void);
 
 #define PqPageStoreTrace DEBUG5
 
-#define ZENITH_TAG "[ZENITH_SMGR] "
-#define zenith_log(tag, fmt, ...) ereport(tag, \
-		(errmsg(ZENITH_TAG fmt, ## __VA_ARGS__), \
+#define NEON_TAG "[NEON_SMGR] "
+#define neon_log(tag, fmt, ...) ereport(tag, \
+		(errmsg(NEON_TAG fmt, ## __VA_ARGS__), \
 		 errhidestmt(true), errhidecontext(true)))
 
 bool		connected = false;
 PGconn	   *pageserver_conn = NULL;
 
 char	   *page_server_connstring_raw;
 
-static ZenithResponse *zenith_call(ZenithRequest *request);
+static ZenithResponse *pageserver_call(ZenithRequest *request);
```

Comment:
Is the renaming of Zenith[Response, Request] excluded from this patch's efforts?

Reply:
Yeah, I left that for another path.